toolchain/lowering/lowering_handle.cpp

```diff
@@ -259,7 +273,17 @@ auto LoweringHandleStructValue(LoweringFunctionContext& context,
 auto LoweringHandleStubReference(LoweringFunctionContext& context,
                                  SemanticsNodeId node_id, SemanticsNode node)
     -> void {
-  context.SetLocal(node_id, context.GetLocal(node.GetAsStubReference()));
+  if (node.type_id() == SemanticsTypeId::TypeType) {
+    if (!context.TryToReuseLocal(node_id)) {
+      llvm::Value* v = llvm::ConstantInt::get(context.builder().getInt1Ty(),
+                                              node.GetAsStubReference().index);
```

Comment:
As you know this is just a workaround and as we are using `Int1Ty` as a placeholder type for TypeType until it is handled appropriately, I am passing the index here instead of 0 or 1 so that if the index is non negative it will get 1.  In the test, the store instruction is set to true if the index is non negative. Which means it will try to write to the memory location either true or false.

Reply:
In the tests, I see a bunch of `i1 true`, no `i1 false`. Can you please either add a test which covers the behavior (and comment the code to explain the significance), or just pass 0?